code/modules/organs/body_modifications.dm

```diff
@@ -234,9 +238,15 @@ var/global/list/modifications_types = list(
 /datum/body_modification/organ/robotize_organ/create_organ(var/mob/living/carbon/holder, O, color)
 	var/obj/item/organ/I = ..(holder,O,color)
 	I.nature = MODIFICATION_SILICON
+	if(istype(I, /obj/item/organ/internal/appendix))
```

Comment:
These appendix checks feel very wrong.

Reply:
its litteraly by the book.

Reply:
It's just that there is no assisted appendix sprite right? Don't you want to just not set `I.icon_state = "[I.icon_state]_assisted"` but still applying the other lines so that it's named assisted appendix and have the right stats?

Reply:
k